src/files.js

```diff
@@ -527,4 +525,20 @@ FOAM_FILES([
   { name: "foam/test/TestObj" },
   { name: "foam/core/FObjectTest" },
   { name: "foam/nanos/dig/exception/DigErrorMessage" },
+
```

Comment:
These should be moved to a support.js file under the support package like nanos. This prevents users of FOAM who don't use the support portal from increasing the size of their FOAM lib.